ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js

```diff
@@ -865,58 +874,58 @@ let getHeadmarkerId = (data, matches) => {
         return {
           en: 'Enumeration: ' + names.map((x) => data.ShortName(x)).join(', '),
           de: 'Enumeration: ' + names.map((x) => data.ShortName(x)).join(', '),
+          fr: 'Énumeration: ' + names.map((x) => data.ShortName(x)).join(', '),
           ja: 'カウント: ' + names.map((x) => data.ShortName(x)).join(', '),
-          fr: 'Enumeration: ' + names.map((x) => data.ShortName(x)).join(', '),
-          ko: '인원수 대상: ' + names.map((x) => data.ShortName(x)).join(', '),
           cn: '计数' + names.map((x) => data.ShortName(x)).join(', '),
+          ko: '인원수 대상: ' + names.map((x) => data.ShortName(x)).join(', '),
         };
       },
     },
     {
       id: 'TEA Limit Cut Shield',
       regex: Regexes.ability({ source: 'Cruise Chaser', id: '4833', capture: false }),
-      regexCn: Regexes.ability({ source: '巡航驱逐者', id: '4833', capture: false }),
       regexDe: Regexes.ability({ source: 'Chaser-Mecha', id: '4833', capture: false }),
-      regexFr: Regexes.ability({ source: 'Croiseur-chasseur', id: '4833', capture: false }),
+      regexFr: Regexes.ability({ source: 'Croiseur-Chasseur', id: '4833', capture: false }),
       regexJa: Regexes.ability({ source: 'クルーズチェイサー', id: '4833', capture: false }),
+      regexCn: Regexes.ability({ source: '巡航驱逐者', id: '4833', capture: false }),
       regexKo: Regexes.ability({ source: '순항추격기', id: '4833', capture: false }),
       delaySeconds: 2,
       infoText: {
         en: 'Break Shield From Front',
         de: 'Schild von Vorne zerstören',
+        fr: 'Brisez le bouclier du devant',
```

Comment:
For me, yes. Jeremy to check so that reassures me because in French, there are a lot of synonyms. this is even disturbing at times. example: Break the shield from the front, from the front, from the front, from the front, from the front. it looks alike, but at the same time misleading. Sorry, myself, at times I find that our language is too complex for too many things. I am going to modify slightly, I just thought otherwise thanks to you.